packages/combo-box/src/vaadin-combo-box-mixin.d.ts

```diff
@@ -145,8 +145,15 @@ export declare class ComboBoxMixinClass<TItem> {
 
   protected readonly _propertyForValue: string;
 
+  protected _dropdownItems: TItem[];
```

Comment:
This property replaces `_items` that was previously used by `vaadin-combo-box-dropdown`. It seems like we can use `filteredItems` instead, so I will try to refactor that in a separate commit in this PR.